go.mod

```diff
@@ -16,6 +17,7 @@ require (
 	github.com/cosmos/gogoproto v1.4.10
 	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7 v7.1.3-0.20240228213828-cce7f56d000b
 	github.com/cosmos/ibc-apps/modules/ibc-hooks/v7 v7.0.0-20240403143657-8e64543c87e0
+	github.com/cosmos/ibc-go/modules/light-clients/08-wasm v0.1.2-0.20240412103620-7ee2a2452b79
```

Comment:
It compiles properly, so I'm okay with this. I guess the way to truly test this is with integration tests.

Reply:
are you referring to the question of whether the version is correct?

Reply:
Whether hacking the commit hash like that is okay

Reply:
oh, it was done by `go get`

Reply:
Yeah I meant instead of getting an "official" tag